copyright

```diff
@@ -1567,6 +1568,24 @@ Copyright: National Archives and Records Administration
 License: public-domain
  Taken from Wikimedia Commons. Cropped and edited.
 
+Files:
+ images/land/badlands14*
+Copyright: Azzedine Rouichi
+License: Unsplash
+Comment:
+ This image was uploaded to
+ unsplash.com after June 2017, so it is subject to the
+ Unsplash License. It was also uploaded after February 2018,
+ so it is subject to an additional restriction limiting the
+ sale of unaltered copies.
+
+Files:
+ images/land/lava0*
+Copyright: USGS
+License: public-domain
+ From US USGS, and therefore in the public domain because it was created by
+ government employees while doing work for the government.
+
```

Comment:
```suggestion

```

There's a duplicated entry in the copyright for this file:
![image](https://github.com/user-attachments/assets/cbdcf27d-2385-4ba8-b1ca-4aec5aa319cc)

They seem to be the same? So just removing one should work.